docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -1798,6 +1851,30 @@ Default stratifications to all observers should include scenario and input draw.
       * Use the interactive sim to confirm there is no coverage of ACS outside of the eligible gestational age range.
     - 
     - 
+  * - 17.0
+    - * Confirm that baseline coverage of IFA matches the health systems team estimates of IFA at ANC as specified in the :ref:`oral iron supplementation intervention page <oral_iron_antenatal>`.
+      * Confirm that baseline coverage of MMS is 0 in all locations.
+      * Confirm that only simulants who attend ANC receive IFA in all scenarios.
+      * Use the interactive sim to confirm that in the baseline scenario, the exposure distribution of hemoglobin and anemia among PLW and WRA as well as the morbidity and mortality rates of each maternal disorder matches that of GBD.
```

Comment:
We don't model WRA, do we?

Also note that we have checked morbidity/mortality of maternal disorders using observers, not the interactive sim. We used the interactive sim to check the risk effect of hemoglobin specifically.